gen-ai/orchestrator-server/src/main/python/server/src/gen_ai_orchestrator/configurations/environment/settings.py

```diff
@@ -96,6 +96,8 @@ class _Settings(BaseSettings):
     observability_proxy_server: Optional[ProxyServerType] = None
     observability_proxy_server_authorization_header_name: Optional[str] = None
 
+    gcp_project_id: Optional[str] = None
```

Comment:
we don't need an alias, because on the orchestrator, we'll pass the gcp project_id with `tock_gen_ai_orchestrator_gcp_project_id`. The `tock_gcp_project_id` will be used on Bot Admin.

Reply:
I was worried that the prefix would also apply to the alias, but that's not the case, so I added it

Reply:
Yes alias will work without the prefix 👍️